### PR TITLE
Squad external config

### PR DESCRIPTION
**Pull request by mwasilew**

### CODE REVIEW

File: squad/api/rest.py [hunk 1]
```diff
@@ -163,6 +164,36 @@ def testjobs(self, request, pk=None):
...
+        output_format = request.query_params.get("output", "text/plain")
+        template_id = request.query_params.get("template", None)
+        template = None
+        if template_id != "-1":  # query_params are strings
+            template = self.get_object().project.custom_email_template
+        if template_id is not None:
+            try:
+                template = EmailTemplate.objects.get(pk=template_id)
+            except EmailTemplate.DoesNotExist:
+                pass
```
Comment (danrue) [inline squad/api/rest.py]: I'm confused here starting at line 180. I suggest adding a test for pass template=-1, and also passing some valid template id. Is passing -1 the same as not specifying a template?

Comment (mwasilew) [reply squad/api/rest.py]: passing -1 will force using 'pre-built' squad templates. Passing nothing as a template will result in using template set for the project (if there is one).

Comment (terceiro) [reply squad/api/rest.py]: I would rather use an explicit placeholder name instead of -1, e.g. `"builtin"`

Comment (mwasilew) [reply squad/api/rest.py]: ok, I'll change it to string

**Pull request merged**